src/java.base/share/classes/jdk/internal/foreign/abi/aarch64/CallArranger.java

```diff
@@ -480,12 +480,10 @@ List<Binding> getBindings(Class<?> carrier, MemoryLayout layout) {
                     StorageCalculator.StructStorage[] structStorages
                             = storageCalculator.structStorages((GroupLayout) layout, forHFA);
 
-                    for (StorageCalculator.StructStorage(
-                            long offset, Class<?> ca, int byteWidth, VMStorage storage
-                    ) : structStorages) {
+                    for (StorageCalculator.StructStorage structStorage : structStorages) {
```

Comment:
This change is not strictly required, but I was getting weird errors when building javadoc from IntelliJ w/o it. Since patterns in for-each are going away anyways, I thought perhaps we could just take care of this.